python/grass/temporal/univar_statistics.py

```diff
@@ -65,11 +65,10 @@ def compute_univar_stats(registered_map_info, stats_module, fs, rast_region=Fals
 
     if not univar_stats:
         gs.warning(
-            _(
-                "Unable to get statistics for {voxel}raster map "
-                "<{rmap}>".format(
-                    rmap=id, voxel="" if stats_module.name == "r.univar" else "3d "
-                )
+            _("Unable to get statistics for raster map <{rmap}>").format(rmap=id)
+            if stats_module.name == "r.univar"
+            else _("Unable to get statistics for 3d raster map <{rmap}>").format(
```

Comment:
```suggestion
            else _("Unable to get statistics for 3D raster map <{rmap}>").format(
```

Reply:
And yes, the original string is wrong, so please keep you changes.

Reply:
I'll need to be on a computer to properly check what changes of the two suggestions to keep.